libs/rbcapi/src/bills/managePayees.ts

```diff
@@ -0,0 +1,111 @@
+import { log } from '@thecointech/logging';
+import { ApiAction } from '../action';
+import { openBillPaymentPage } from './index';
+
+const Errors = {
+  MissingAccountNumber: "MISSING-ACCOUNT-NUMBER",
+  MissingAccountNickname: "MISSING-ACCOUNT-NICKNAME",
+  DuplicateAccountNumber: "DUPLICATE-ACCOUNT-NUMBERS",
+  SetAccountNumberFailed: "SET-ACC-NO-FAILED",
+  DuplicateNameError: "DUPLICATE-NAME-ERROR",
+  MissingLink: "MISSING-LINK",
+}
+
+//
+// Add a new payee and set it's nickname.  Once done, return to the main page.
+export async function addPayee(act: ApiAction, nickname: string, payee: string, accountNumber: string) {
```

Comment:
I'm sure there is a better way to do web scraping, rather than relying on all these brittle IDs.

It seems like a perfect job for an neural net.  That would be a helluva product (next year)